toolkits/global/packages/global-facet/scss/10-settings/default.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * Facet
+ * Default branding for component
+ */
+
+$facet--background-color: lightgrey;
+$facet--color: black;
+$facet--font-bold: 700;
+$facet--font-size-small: 1.4rem;
+$facet--font-size: 1.6rem;
+$facet--spacing-x-small: 4px;
```

Comment:
We have a default spacing map in brand context so it's probably worth defining these using the spacing function .e.g. `spacing(4)`

```$context--spacing: (0, 2, 4, 8, 16, 24, 32, 48, 64);```